DashWallet/Sources/Models/CrowdNode/CrowdNode.swift

```diff
@@ -62,6 +62,8 @@ public class CrowdNodeObjcWrapper: NSObject {
 
 private let kValidStatus = "valid"
```

Comment:
@Syn-McJ I see that we are passing the status as a string, but since we have a predefined statuses it would be better to have a string enum.

Reply:
The `OnlineAccountState` and these statuses are different - we have the first to keep the state of the app, while these are just constants to check which response had CrowdNode API returned. 

I don't think we should mix them since that'd could be confusing.